README.md

```diff
@@ -177,4 +177,16 @@ As mentioned below in best practices please _make sure your tests are always iso
 - Don't use selectors which are highly likely to change such as react id's rather use data attributes.
 - Don't use the previous test result for granted for next result. If the login passes in previous test you should login again for the new test even if it takes slightly more time.
 - Don't do things like `cy.wait(5000 seconds)` these are arbitrary and prone to failure depending on request and execution time.
-- You can add the constants and files on `cypress/fixtures` — for example, the `cypress/fixtures/sample.csv` which can be used in commands and tests.
\ No newline at end of file
+- You can add the constants and files on `cypress/fixtures` — for example, the `cypress/fixtures/sample.csv` which can be used in commands and tests.
+
+#### Non-browser based integration test
+
+There is also a script to use [CKAN JavaScript client](https://github.com/datopian/ckan-client-js) to run in integration test:
+
+```console
+$ node ntegration_test_with_ckan_client.js <ORGANIZATION NAME>
+```
+
+This script requires a CKAN instance set with an organization.
```

Comment:
What does this script do? I assume it auto creates a dataset, resource and then uploads file. Let's mention that and make sure his information it output by the script (e.g. urls are output to dataset and resource).